script/ts-proto.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+TMP_GRPC_JS_SERVICES_DIR="${AKASH_DEVCACHE_BASE}/ts/generated-grpc-js"
+
+function prepare_grpc_js() {
+    mkdir -p "$TMP_GRPC_JS_SERVICES_DIR"
+}
+
+function add_grpc_js_services() {
+  local ts_grpc_js_services
+  local file
+  local dest_path
+  local index_file_path
+  local index_file_name
+  local export_statement
+
+  ts_grpc_js_services=$(find "$TMP_GRPC_JS_SERVICES_DIR" -name 'service.ts')
+
+  for file in $ts_grpc_js_services; do
+    dest_path=$(dirname "${file/$TMP_GRPC_JS_SERVICES_DIR/$AKASH_TS_ROOT\/src\/generated}")
+    dest_file="${dest_path}/service.grpc-js.ts"
+
+    mv "$file" "$dest_file"
+
+    path_from_gen_dir=${dest_file#"${AKASH_TS_ROOT}/src/generated/"}
+    index_file_name_base=${path_from_gen_dir%/service.grpc-js.ts}
+    index_file_name="index.${index_file_name_base//\//.}.grpc-js.ts"
+    index_file_path="${AKASH_TS_ROOT}/src/generated/$index_file_name"
+    export_statement="export * from \"./${path_from_gen_dir%.ts}\";"
+
+    echo "$export_statement" > "$index_file_path"
+  done
+
+  rm -rf "$TMP_GRPC_JS_SERVICES_DIR"
```

Comment:
lets put this inside `cleanup` function once this part is moved to `protocgen-legacy.sh`